fuzzing/fuzzer.go

```diff
@@ -841,6 +853,10 @@ func (f *Fuzzer) Start() error {
 			f.logger.Info("Coverage report saved to file: ", colors.Bold, coverageReportPath, colors.Reset)
 		}
 	}
+	err = f.ReversionStats.WriteReport(f.config.Fuzzing.CorpusDirectory, f.logger)
```

Comment:
Should this check whether it's enabled?

Reply:
negative, I let the ReversionStats object keep track of whether it's supposed to be enabled to improve decoupling:

https://github.com/crytic/medusa/blob/79ccf38be53501bee63f1252bd0ef8141c2ac694/fuzzing/reversion/reversion_statistics.go#L117-L119